app/javascript/packs/MainApp/constants.js

```diff
@@ -0,0 +1,24 @@
+const currencyConfig = [
+  {
+    total: '23',
+    currency: 'AUD'
+  },
+  {
+    total: '23',
+    currency: 'CAD'
+  },
+  {
+    total: '15',
+    currency: 'EUR'
+  },
+  {
+    total: '10',
+    currency: 'GBP'
+  },
+  {
+    total: '16',
+    currency: 'USD'
+  }
+]
+
+export default currencyConfig
```

Comment:
How about putting the certification constants ('snitch', 'assistant', 'head', and 'field') in this file too if we’re already calling it constants.js? Currently we’re having them as strings in the source code which isn’t that great.

Reply:
Good point, will add.